tests/unit_tests/storage/test_subject_manager.py

```
@@ -0,0 +1,504 @@
+        sub_storage = updater._sub_agent_storage_ext_knowledge(
+            __import__("datus.schemas.agent_models", fromlist=["SubAgentConfig"]).SubAgentConfig(
+                system_prompt=sub_agent_name
+            )
+        )
+        _seed_ext_knowledge(sub_storage)
+
+        updater.update_ext_knowledge(SUBJECT_PATH, EXT_KNOWLEDGE_NAME, {"explanation": "sub-agent updated"})
+
+        # Verify main storage was updated
+        main_results = updater.ext_knowledge_storage.search_knowledge(
+            subject_path=SUBJECT_PATH + [EXT_KNOWLEDGE_NAME], top_n=None
```

Ruff (0.15.2)

[warning] Consider `[*SUBJECT_PATH, EXT_KNOWLEDGE_NAME]` instead of concatenation

Replace with `[*SUBJECT_PATH, EXT_KNOWLEDGE_NAME]`

(RUF005)